src/js/player.js

```diff
@@ -2471,11 +2471,7 @@ class Player extends Component {
         this.techCall_('src', source.src);
       }
 
-      if (this.options_.preload === 'auto') {
-        this.load();
-      }
-
-    // Set the source synchronously if possible (#2326)
+      // Set the source synchronously if possible (#2326)
```

Comment:
this indent is interesting. I think it was dedented so that it lines up with the this.ready call, not sure if it's necessary or not. Maybe we want to make a new line for the arg and have the comment directly above the arg?